### PR TITLE
OSM extract GIS layers doc

### PR DESCRIPTION
**Pull request by ebocher**

### CODE REVIEW

**Comment by gpetit** [comment]
I will check your PR when you validate mine ;-)

File: docs/chain_documentation/workflow/osm/data_preparation.md [hunk 1]
```diff
@@ -1,6 +1,68 @@
 # OSM  - Extraction and transformation to GIS layers
 
+Figure 1 shows the sequence of actions that are performed to retrieve OSM data and transform them into GIS layers.
+
+![osm_extraction](../../../resources/images/chain_documentation/osm_extract_gis_layers.png)
+
+Figure 1. OSM data flow transformation
+
+The sequence starts either by specifying a place name or by specifying an OSM bbox.
```
Comment (gpetit) [inline docs/chain_documentation/workflow/osm/data_preparation.md]: Why are you talking about an OSM bbox. It's a bbox ?!
We say place name or placename ?

File: docs/chain_documentation/workflow/osm/data_preparation.md [hunk 2]
```diff
@@ -1,6 +1,68 @@
...
+
+If the user set a place name, the [Nominatim API](https://nominatim.org) is requested  to search in the OSM database all
+objects that match the given text.
```
Comment (gpetit) [inline docs/chain_documentation/workflow/osm/data_preparation.md]: remove the break line

File: docs/chain_documentation/workflow/osm/data_preparation.md [hunk 3]
```diff
@@ -1,6 +1,68 @@
...
+	way;
+	relation;
+);
+(._;>;);
+out;
+```
+
+The response of the query is stored in an OSM xml file. 
+The XML file is loaded in a relational spatial database (H2GIS or PostGIS).
+Thanks to the [H2GIS OSM driver](http://www.h2gis.org/docs/1.5.0/OSMRead/) that performs this task.
```
Comment (gpetit) [inline docs/chain_documentation/workflow/osm/data_preparation.md]: No point after `task`

File: docs/chain_documentation/workflow/osm/data_preparation.md [hunk 4]
```diff
@@ -1,6 +1,68 @@
...
+out;
+```
+
+The response of the query is stored in an OSM xml file. 
+The XML file is loaded in a relational spatial database (H2GIS or PostGIS).
+Thanks to the [H2GIS OSM driver](http://www.h2gis.org/docs/1.5.0/OSMRead/) that performs this task. 
+1O tables are produced. They represent the whole OSM data model (node, way, relation, node_tag...).
+
+
+Once the 10 tables have been created a set of SQL scripts are run to transform the OSM data into GIS layers.
```
Comment (gpetit) [inline docs/chain_documentation/workflow/osm/data_preparation.md]: have been created,

File: docs/chain_documentation/workflow/osm/data_preparation.md [hunk 5]
```diff
@@ -1,6 +1,68 @@
...
+
+Once the 10 tables have been created a set of SQL scripts are run to transform the OSM data into GIS layers. 
+These scripts are grouped to offer 3 kind of transformations : 
+- point geometries,
+- line geometries,
+- polygon geometries
+
+For each transformation, it is possible to specify : 
+- the epsg code of the output geometries
+- a list of OMS keys to filter the OSM objects. e.g ["building", "water", "landuse"]
```
Comment (gpetit) [inline docs/chain_documentation/workflow/osm/data_preparation.md]: OMS ?

File: docs/chain_documentation/workflow/osm/data_preparation.md (hunk 1 above)
Comment (ebocher) [reply docs/chain_documentation/workflow/osm/data_preparation.md]: osm bbox is a specific bbox

Comment (ebocher) [reply docs/chain_documentation/workflow/osm/data_preparation.md]: all system do not use the same bbox

Comment (gpetit) [reply docs/chain_documentation/workflow/osm/data_preparation.md]: You mean in terms of coordinate system ? Or the way we list the coordinates ?

File: docs/chain_documentation/workflow/osm/data_preparation.md (hunk 3 above)
Comment (ebocher) [reply docs/chain_documentation/workflow/osm/data_preparation.md]: don't understand

File: docs/chain_documentation/workflow/osm/data_preparation.md (hunk 5 above)
Comment (ebocher) [reply docs/chain_documentation/workflow/osm/data_preparation.md]: jajaja

File: docs/chain_documentation/workflow/osm/data_preparation.md (hunk 3 above)
Comment (gpetit) [reply docs/chain_documentation/workflow/osm/data_preparation.md]: The sentence is not finished, so no point after the word "task"

File: docs/chain_documentation/workflow/osm/data_preparation.md (hunk 1 above)
Comment (ebocher) [reply docs/chain_documentation/workflow/osm/data_preparation.md]: the order of the lat, lon are not the same

Comment (ebocher) [reply docs/chain_documentation/workflow/osm/data_preparation.md]: placename is the identifier in code.

File: docs/chain_documentation/workflow/osm/data_preparation.md (hunk 3 above)
Comment (gpetit) [reply docs/chain_documentation/workflow/osm/data_preparation.md]: Thanks to the [H2GIS OSM driver](http://www.h2gis.org/docs/1.5.0/OSMRead/) that performs this task, 10 tables are produced.

**Pull request merged**